index.js

```diff
@@ -31,10 +31,5 @@ module.exports = function (hex) {
 
 	const num = parseInt(hex, 16);
 	// eslint-disable-next-line no-mixed-operators
```

Comment:
Better to fix the violation than ignoring: https://eslint.org/docs/rules/no-mixed-operators